src/components/pages/Weather/Weather.js

```diff
@@ -1,11 +1,57 @@
 import React from 'react';
+import firebase from 'firebase/app';
 import './Weather.scss';
+import connection from '../../../helpers/data/connection';
+import getWeather2 from '../../../helpers/data/weatherRequests';
+import CurrentWeather from './CurrentWeather';
+import WeatherForm from './WeatherForm';
+
 
 class Weather extends React.Component {
+  state = {
+    uid: '',
+    weatherArray2: [],
+  }
+
+  getWx = (uid2) => {
```

Comment:
Why have you written this function? Are you going to use it to implement code in the future?

Reply:
This was the only way  I could see to populate the weather Array and set it to state because I wasn't sure how to pass the state from App.js to Weather.js if it was to call in App.js. should I be getting that same in formation via the componentWillMount?